tests/test_flask_util.py

```diff
@@ -303,6 +309,19 @@ def test_callback_view_errors(self):
             rv = c.get('/oauth2callback?state=%s&code=codez' % state)
             self.assertEqual(rv.status_code, httplib.BAD_REQUEST)
 
+        # Missing flow.
+        with self.app.test_client() as c:
+            with c.session_transaction() as session:
+                session['google_oauth2_csrf_token'] = 'tokenz'
+
+            state = json.dumps({
+                'csrf_token': 'tokenz',
+                'return_url': '/return_url'
+            })
+
+            rv = c.get('/oauth2callback?state=%s&code=codez' % state)
+            self.assertEqual(rv.status_code, httplib.BAD_REQUEST)
```

Comment:
Feeling a little cheeky I would also be satisfied with `assertIs` here rather than `assertEqual`. Your call.